packages/main/src/plugin/agent-registry.ts

```
@@ -0,0 +1,131 @@
+    }, []);
+    const result = [];
+    for (const modelType of modelTypes) {
+      if (await isSupportedModelType({ name: modelType })) {
+        result.push({ name: modelType });
+      }
+    }
+    return result;
+  }
+
+  invalidate(): void {
+    this.agentRegistrations.values().forEach(agentRegistration => (agentRegistration.agentInfo = undefined));
```

Biome (2.4.15)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)